.github/renovate.json

```diff
@@ -5,5 +5,7 @@
   "baseBranches": [
     "release/v2.7"
   ],
-  "prHourlyLimit": 2
+  "prHourlyLimit": 2,
+  "prConcurrentLimit": 4,
+  "branchConcurrentLimit": 4
```

Comment:
Since rancher/rancher has 2-4 active branches, how will these two items interact with each other ? The docs don't make sense to me.

> If you want the same limit for both concurrent branches and concurrent PRs, then just set a value for prConcurrentLimit and it will be reused for branch calculations too. But if you want to allow more concurrent branches than concurrent PRs, you can configure both values (e.g. branchConcurrentLimit=5 and prConcurrentLimit=3).

https://docs.renovatebot.com/configuration-options/#branchconcurrentlimit

The way I read that it means if you have 5 branches and 5 prs, with a branchLimit of 5, than each branch only gets 1 PR ?

Reply:
For [renovate purposes](https://github.com/rancher/rancher/blob/d24d24c7b10bdf9556ce4fd5ed6cdd415beb3781/.github/renovate.json#L6), we are currently only tracking `release/v2.7`, so this would limit to 4 PRs for that base branch alone.

Please note I have removed the `branchConcurrentLimit` value, as that was redundant.